pkgs/misc/emulators/melonDS/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, SDL2, gtk3, curl, libpcap }:
+
+stdenv.mkDerivation {
```

```suggestion
stdenv.mkDerivation rec {
```

Forgot you need this here.